### PR TITLE
Update HoVerNet pipeline

### PR DESCRIPTION
**Pull request by KumoLiu**
Fixes #.

### Description
add training mode descriptions in README
update prepare_patches.py
change to "fast" mode

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

File: pathology/hovernet/README.MD [hunk 1]
```diff
@@ -47,6 +47,8 @@ python ./prepare_patches.py \
 This example uses MONAI workflow to train a HoVerNet model on prepared CoNSeP dataset.
 Since HoVerNet is training via a two-stage approach. First initialized the model with pre-trained weights on the [ImageNet dataset](https://ieeexplore.ieee.org/document/5206848), trained only the decoders for the first 50 epochs, and then fine-tuned all layers for another 50 epochs. We need to specify `--stage` during training.
 
+There are two training modes in total. If "original" mode is specified, using [270, 270] and [80, 80] for `patch_size` and `out_size` respectively. If "fast" mode is specified, using [256, 256] and [164, 164] for `patch_size` and `out_size` respectively. The results we show below are based on the "fast" model.
```
Comment (drbeh) [inline pathology/hovernet/README.MD:50]:
```suggestion
There are two training modes in total. If "original" mode is specified, it uses [270, 270] and [80, 80] for `patch_size` and `out_size` respectively. If "fast" mode is specified, it uses [256, 256] and [164, 164] for `patch_size` and `out_size` respectively. The results we show below are based on the "fast" model.
```

File: pathology/hovernet/README.MD [hunk 2]
```diff
@@ -78,7 +80,7 @@ torchrun --nnodes=1 --nproc_per_node=2 training.py --stage 1
 #### [HoVerNet Validation](./evaluation.py)
 
 This example uses MONAI workflow to evaluate the trained HoVerNet model on prepared test data from CoNSeP dataset.
-With their metrics on original mode. We reproduce the results with Dice: 0.82762; PQ: 0.48976; F1d: 0.73592.
+With their metrics on fast mode. We reproduce the results with Dice: 0.83285; PQ: 0.49765; F1d: 0.74214.
```
Comment (drbeh) [inline pathology/hovernet/README.MD:83]:
```suggestion
Using this training pipeline in the "fast" mode, we have achieved the following metrics: Dice: 0.8329, PQ: 0.4977 and F1d: 0.7421.
```

File: pathology/hovernet/hovernet_training_torch.ipynb [hunk 3]
```diff
@@ -23,7 +23,7 @@
     "\n",
     "The CoNSeP dataset comes from https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet/.\n",
     "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Project-MONAI/tutorials/blob/main/pathology/hovernet/training_torch.ipynb)"
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Project-MONAI/tutorials/blob/main/pathology/hovernet/hovernet_training_torch.ipynb)"
```
Comment (drbeh) [inline pathology/hovernet/hovernet_training_torch.ipynb:26]: Maybe changing the name of the notebook to `hovernet_torch` as it already includes training, validation and also sliding inference. what do you think?

File: pathology/hovernet/prepare_patches.py [hunk 4]
```diff
@@ -16,6 +16,10 @@ def load_img(path):
 
 
 def load_ann(path):
+    """
+    This function is specific to CoNSeP dataset.
+    if own dataset is used, then below may need to be modified
```
Comment (drbeh) [inline pathology/hovernet/prepare_patches.py:21]:
```suggestion
    if using other datasets, the code below may need to be modified.
```

**Review by drbeh — COMMENTED** [review]
HI @KumoLiu, it looks good to me. just few comments of the texts.

File: pathology/hovernet/README.MD (hunk 1 above)
Comment (KumoLiu) [reply pathology/hovernet/README.MD]: Have updated, thanks!

File: pathology/hovernet/hovernet_training_torch.ipynb (hunk 3 above)
Comment (KumoLiu) [reply pathology/hovernet/hovernet_training_torch.ipynb:26]: Yes, I think so, have renamed the file name! Thanks!

**Comment by drbeh** [comment]
@KumoLiu, could you please update the default mode of other hovernet pipelines too? thanks
https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/hovernet/evaluation.py#L142
https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/hovernet/inference.py#L189

**Comment by KumoLiu** [comment]
> @KumoLiu, could you please update the default mode of other hovernet pipelines too? thanks
> 
> https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/hovernet/evaluation.py#L142
> 
> 
> https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/hovernet/inference.py#L189

Of course, have updated!

**Comment by drbeh** [comment]
@KumoLiu this one needs to be fixed too. Thanks
https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/hovernet/inference.py#L136

**Comment by KumoLiu** [comment]
Hi @drbeh, I have already updated the performance curve, could you please double-check the license descriptions in the README, if that's ok I think this PR can be merged, thanks!

**Review by drbeh — APPROVED** [review]

**Comment by wyli** [comment]
in the readme and notebookt I think we should cite this paper https://www.sciencedirect.com/science/article/abs/pii/S1361841519301045 instead of the arxiv version.

do we need an acknowledgement section for the original author/implementations? the detection folder's readme is a good example https://github.com/Project-MONAI/tutorials/tree/main/detection#acknowledgement cc @drbeh @JHancox

**Comment by drbeh** [comment]
> in the readme and notebookt I think we should cite this paper https://www.sciencedirect.com/science/article/abs/pii/S1361841519301045 instead of the arxiv version.
> 
> do we need an acknowledgement section for the original author/implementations? the detection folder's readme is a good example https://github.com/Project-MONAI/tutorials/tree/main/detection#acknowledgement cc @drbeh @JHancox

I agree with @wyli!

@KumoLiu would it be possible to update them using the following citation? thanks

Simon Graham, Quoc Dang Vu, Shan E Ahmed Raza, Ayesha Azam, Yee Wah Tsang, Jin Tae Kwak, Nasir Rajpoot, Hover-Net: Simultaneous segmentation and classification of nuclei in multi-tissue histology images, Medical Image Analysis, 2019
https://doi.org/10.1016/j.media.2019.101563

**Pull request merged**